### PR TITLE
oma 2.4.1

### PR DESCRIPTION
**Pull request by alpae**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

**Comment by alpae** [comment]
not sure I understand the problem. from the log in brewsci.homebrew-bio it seems that the installation and running tests were successful - this is also what I observe on linuxbrew docker image. To me it it is not clear why there is an error reported.

File: Formula/oma.rb [hunk 1]
```diff
@@ -15,10 +16,28 @@ class Oma < Formula
...
+  end
+
+  resource "lxml" do
+    url "https://files.pythonhosted.org/packages/e4/19/8dfeef50623892577dc05245093e090bb2bab4c8aed5cad5b03208959563/lxml-4.4.2.tar.gz"
+    sha256 "eff69ddbf3ad86375c344339371168640951c302450c5d3e9936e98d6459db06"
+  end
+
   def install
-    system "./install.sh", prefix, share
+    venv = virtualenv_create(libexec, python="python3")
```
Comment (jonchang) [inline Formula/oma.rb]:
```suggestion
    venv = virtualenv_create(libexec, "python3")
```

Comment (alpae) [reply Formula/oma.rb]: thanks @jonchang , helped at least for macos.

**Comment by alpae** [comment]
@jonchang ok, seems to finally worked out on both platforms. Let me know if I need to squash these series of commits in order to merge the PR or if this is fine to stay. Thanks for your help on this again!

**Comment by jonchang** [comment]
I see that this formula now `depends_on` libxml2 and libxslt; these libraries ship with macOS. Can you confirm that these are necessary on macOS? If they’re not, please replace them with `uses_from_macos` lines instead. 

Please rebase and squash into a single commit following Homebrew style, thanks.

**Comment by alpae** [comment]
@jonchang, I've adjusted the dependencies to uses_from_macos for libxml2 and libxslt, and rebased into a single commit the changes of the oma formula.

**Comment by jonchang** [comment]
Merged. Thanks!

**Pull request closed without merging**